<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDoc</class>
 <widget class="QDialog" name="HelpDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Visit &lt;a href=&quot;http://taki.sourceforge.net/&quot;&gt;Taki website&lt;/a&gt; for the latest Taki news, updates, and support. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTextBrowser" name="manBrowser">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse</set>
       </property>
       <property name="source">
        <url>
         <string>qrc:/doc/manual.html</string>
        </url>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frBBControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QLabel" name="bbFrameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btCancelBB">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btBackBB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frDesktopControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <widget class="QPushButton" name="btCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btBack">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btCancel</sender>
   <signal>pressed()</signal>
   <receiver>HelpDoc</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancelBB</sender>
   <signal>clicked()</signal>
   <receiver>HelpDoc</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btBackBB</sender>
   <signal>clicked()</signal>
   <receiver>manBrowser</receiver>
   <slot>backward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btBack</sender>
   <signal>clicked()</signal>
   <receiver>manBrowser</receiver>
   <slot>backward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manBrowser</sender>
   <signal>backwardAvailable(bool)</signal>
   <receiver>HelpDoc</receiver>
   <slot>onBackwardAvailable(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>logMsgChanged(int)</slot>
  <slot>onBackwardAvailable(bool)</slot>
 </slots>
</ui>
